lib/libyui_client/widgets/multilinebox.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module LibyuiClient
+  module Widgets
+    # Class representing a multiline box in the UI. It can be YMultiLineEdit.
+    class Multilinebox < Widgets::Base
+      # Returns maximum string length to set in the multiline box
```

Comment:
this one is neatpicking, but I can see that everyone is writing it in free style :) because there are many ways to write UI control names. I'd recommend to stick to our framework and how we are going to call them from outside (in `app.rb`) so we could use `multilinebox` instead of `multiline box` and follow this pattern for the rest of the changes.

Reply:
I agree with you, for consistency. Done